src/analytics/Events.tsx

```diff
@@ -663,6 +663,13 @@ export enum JumpstartEvents {
   jumpstart_claim_loading_dismissed = 'jumpstart_claim_loading_dismissed',
   jumpstart_claim_error_dismissed = 'jumpstart_claim_error_dismissed',
   jumpstart_claim_error_contact_support = 'jumpstart_claim_error_contact_support',
+  jumpstart_reclaim_press = 'jumpstart_reclaim_press',
+  jumpstart_reclaim_start = 'jumpstart_reclaim_start',
+  jumpstart_reclaim_failed = 'jumpstart_reclaim_failed',
+  jumpstart_reclaim_succeeded = 'jumpstart_reclaim_succeeded',
+  jumpstart_reclaim_dismiss_error = 'jumpstart_reclaim_dismiss_error',
+  jumpstart_reclaim_fetching_success = 'jumpstart_reclaim_fetching_success',
```

Comment:
what do you think if we add the word `status` here to make it more explicit what this fetching is about?